api/v1/tetragon/tetragon.proto

```diff
@@ -285,3 +285,23 @@ message ProcessLoader {
 	string path = 2;
 	bytes buildid = 3;
 }
+
+// RuntimeHookRequest synchronously propagates information to the agent about run-time state.
+message RuntimeHookRequest {
+    oneof event {
+        CreateContainer createContainer = 1;
+    }
+}
+
+message RuntimeHookResponse {}
+
+// CreateContainer informs the agent that a container was created
+message CreateContainer {
+    // cgroupPath is the cgroup path for the container
+    string cgroupPath = 1;
```

Comment:
Can you past an example of the received cgroup path here? the received info

Reply:
```
createContainer:{cgroupPath:\"/kubepods/besteffort/podb5f5a97e-b11b-416f-8bb3-722b4aa961a1/c7f5120dcae962bfea17a8ad5d5302f567f2ae35103aedc943feb9b69c3a0ce9\" rootDir:\"/run/containerd/io.containerd.runtime.v2.task/k8s.io/c7f5120dcae962bfea17a8ad5d5302f567f2ae35103aedc943feb9b69c3a0ce9/rootfs\"
```